pylint/utils/file_state.py

```diff
@@ -139,6 +139,8 @@ def handle_ignored_message(
         or globally.
         """
         if state_scope == MSG_STATE_SCOPE_MODULE:
+            if not isinstance(line, int):
```

Comment:
Similar problem here: if `state_scope == MSG_STATE_SCOPE_MODULE` then `line` is never not an `int`. How to fix this...?

Reply:
Isn't it an issue to fix in astroid ?

Reply:
No this is a pylint issue. If we disable something via `pylintrc` its disabled on the "package" scope, that's why it doesn't have a line number. If we disable something in a module itself it will always have an assosciated line number

Reply:
All right, thank you for clearing that up.

Maybe we could create a different structure to handle module and package disable, but I'm not very clear on the state of the code right now. I'll have a look before 2.13.0 is released. (Or I trust you if you find a solution that you think is good enough in the meantime).

Reply:
Same here. Either `cast` or possibly `assert`.